Library/Core/UnoCore/Targets/iOS/@(Project.Name).xcodeproj/project.pbxproj

```diff
@@ -458,10 +458,31 @@
 #if @(LIBRARY:Defined)
                 MACH_O_TYPE = mh_dylib;
 #endif
+
+#if @(Project.iOS.SpecialCapabilities.CPPImport:IsSet)
+    #if @(Project.iOS.SpecialCapabilities.CPPImport:Test(1,0))
+                OTHER_CPLUSPLUSFLAGS = (
+					"$(OTHER_CFLAGS)",
+					"-fmodules",
+					"-fcxx-modules",
+				);
+    #endif
+#endif
+#if @(Project.iOS.SpecialCapabilities.AllowSwiftModules:IsSet)
+    #if @(Project.iOS.SpecialCapabilities.AllowSwiftModules:Test(1,0))
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
+    #endif
+#endif
+#if @(Project.iOS.SpecialCapabilities.AllowNonModularModules:IsSet)
+    #if @(Project.iOS.SpecialCapabilities.AllowNonModularModules:Test(1,0))
+                CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES;
+    #endif
+#endif
```

Comment:
...but another question is, do we really need to teach the uno compiler about every possible build-setting? Wouldn't it be easier to add something like 

@(Project.iOS.CustomBuildSettings:Join('\n'))

...so the user can specify these themselves without us having to approve every setting?

Reply:
You're right, but it's highly recommended to let this go as we need to ensure that xcode project file will be correct and fine.
By this way, we don't need to have extra validation for the rules because it predefined strong value either (true) or (false).
Please advice 👍

Reply:
My knee-jerk reaction is that I think I'd prefer throwing a wider net, by providing a weaker but generally more useful mechanism.

That being said, I totally see the point in having strong, less error-prone switches. I'm not really sure what the right balance is here...

Reply:
what we should do right now, @kusma  ?
>I continue creating PRs from our development environments to match all Fuse developers needs and we have advance and larger apps day by day.

Reply:
@devadiab: I'm willing to merge either this (but perhaps with the ususual SpecialCapabilities-namespace removed), or a more generic (but "unsafe") solution.

Reply:
I also suspect that we should add some documentation somewhere?

Reply:
About namespace, take your time and pick a good one okay?
And we can create another PRs for documentation or i can create another commit with documentation. seems good ?

Reply:
@devadiab: What I've been trying to hint at already (perhaps not clearly enough), is that I think directly in the `Project.iOS.*`-namespace would be fine. That would mean, for example `Project.iOS.CPPImport`.

Reply:
Why not, its a good idea; let me change

Reply:
@devadiab: Any chance of this happening? Seems you haven't pushed out such a change yet...